projects/python-examples/algokit_utils_py_examples/transactions/atomic_transaction_groups.py

```diff
@@ -0,0 +1,82 @@
+from algokit_utils import AlgoAmount, PaymentParams
+
+from algokit_utils_py_examples.helpers import setup_localnet_environment
+
+
+def atomic_transaction_groups() -> None:
+    # example: ATOMIC_TRANSACTION_GROUP
+
+    algorand_client, dispenser, account1, account2, account3 = (
+        setup_localnet_environment()
+    )
+
+    """
+    Create a transaction group that will execute atomically
+    Either all transactions succeed, or they all fail
+    """
+    algorand_client.new_group().add_payment(  # First transaction: Payment from A to B
+        PaymentParams(
+            sender=account1.address,
+            receiver=account2.address,
+            amount=AlgoAmount(algo=1),
+            note=b"First payment in atomic group",
+        )
+    ).add_payment(  # Second transaction: Payment from B to C
+        PaymentParams(
+            sender=account2.address,
+            receiver=account3.address,
+            amount=AlgoAmount(algo=2),  # B sends half of what they received to C
+            note=b"Second payment in atomic group",
+        )
+    ).send()  # Send the atomic group of transactions
+
+    """
+    Create a transaction group to simulate
+    Simulation allows you to test the transaction without committing it to the chain
+    """
+    group = (
+        algorand_client.new_group()
+        .add_payment(
+            PaymentParams(
+                sender=account1.address,
+                receiver=account2.address,
+                amount=AlgoAmount(algo=1),
+            )
+        )
+        .add_payment(
+            PaymentParams(
+                sender=account2.address,
+                receiver=account3.address,
+                amount=AlgoAmount(algo=1),
+            )
+        )
+    )
```

Comment:
Missing the closing example comment

```suggestion
        )
    )
    # example: ATOMIC_TRANSACTION_GROUP
```